PowerUp/app/src/main/AndroidManifest.xml

```diff
@@ -18,35 +19,44 @@
         </activity>
         <activity
             android:name=".GameActivity"
-            android:label="@string/app_name" />
+            android:label="@string/app_name"
+            android:theme="@style/MyNoActionBar"/>
         <activity
             android:name=".AvatarRoomActivity"
-            android:label="@string/app_name" />
+            android:label="@string/app_name"
+            android:theme="@style/MyNoActionBar"/>
         <activity
             android:name=".AvatarActivity"
-            android:label="@string/app_name" />
+            android:label="@string/app_name"
+            android:theme="@style/MyNoActionBar"/>
         <activity
             android:name=".MapActivity"
             android:label="@string/app_name"
-            android:launchMode="singleTop" />
+            android:launchMode="singleTop"
+            android:theme="@style/MyNoActionBar"/>
```

Comment:
Is this required in all activities? Change theme of only those as indicated in the issue.